deps/rabbit/src/rabbit.erl

```diff
@@ -175,6 +175,12 @@
                     {requires,    [rabbit_alarm, guid_generator]},
                     {enables,     core_initialized}]}).
 
+-rabbit_boot_step({rabbit_queue_member_eval,
+                   [{description, "Queue membership evaluation"},
```

Comment:
I'm afraid this does not explain what kind of evaluation this boot step performs, and that it only affects QQs.